src/material/core/tokens/m2/mat/_stepper.scss

```diff
@@ -10,7 +10,10 @@ $prefix: (mat, stepper);
 // Tokens that can't be configured through Angular Material's current theming API,
 // but may be in a future version of the theming API.
 @function get-unthemable-tokens() {
-  @return ();
+  @return (
+    // Border radius for step header on hover
+    header-hover-border-radius: 0,
```

Comment:
I think that MDC calls these sorts of tokens as "shape". Also it calls out "state" specifically. E.g. `header-hover-state-shape`.